### DISCUSSION TITLE
Hiding or reordering files and folders in the file tree

### DISCUSSION THREAD

**Discussion by mxgrn** [KosmosisDire/obsidian-webpage-export#337] [Ideas]
Will I be able to hide certain files/folders from the tree, like Obsidian Publish allows me to?

**Comment by KosmosisDire** [comment]
This is a feature I do plan on implementing. In other words, to include the file in the export but not show it in the tree. Mainly I want to think of a clean way to allow the user to specify which files to hide. I don't want to make the options overwhelming so if you have any ideas I'd love to hear them.

**Reply by mxgrn** [reply]
One way could be to copy Obsidian Publish:

<img width="476" alt="image" src="https://github.com/KosmosisDire/obsidian-webpage-export/assets/33935/f746db73-765a-4de6-a6f6-90d7bbf92449">
<img width="452" alt="image" src="https://github.com/KosmosisDire/obsidian-webpage-export/assets/33935/1f996ae5-a0e3-4ea5-a16f-879cba65d5b7">

**Comment by KosmosisDire** [comment]
What do you think about this? This isn't something that exists yet, just a mockup. But the handles on the left would let you reorder and the eye icon on the right would let you hide from the filetree.

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/39423700/0a146458-e1ce-4ade-9172-3c3cd60cb24f)
![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/39423700/8d770023-f927-44ff-aef3-c5c073667b3d)

Downside to this is it condenses a lot of control into one location (which is good and bad)

**Reply by mxgrn** [reply]
Personally, I think it's AWESOME! Way better than Obsidian's. ❤️❤️❤️

**Reply by mxgrn** [reply]
> Downside to this is it condenses a lot of control into one location 

I don't see this as cluttered at all, looks great.

**Reply by KosmosisDire** [reply]
Glad you think so, I will probably go with something similar to this when I do implement the feature.

**Reply by bzetu** [reply]
I also need this function, when can this function be launched, I am looking forward to it @KosmosisDire

**Reply by bzetu** [reply]
It is recommended to use configuration files to implement the hidden function.   Users can fill in the path (file or folder) to be hidden themselves, define a syntax to filter specified prefixes or suffixes.   For example, in the same folder, there is a file named class_cxxx and several files named function_yyy.   This way, we can filter out all files starting with function_
